lab1.js

```diff
@@ -134,6 +159,22 @@ var tooHungryDay;
 */
 
 // your code goes here
+var totalMeals = 0;
+var dateCount = 0;
+var mealsPerDayCounter = mealsPerDay.slice(0);
+
+for (i = 0; i < mealsPerDay.length; i++) {
+	totalMeals += mealsPerDayCounter.shift();
+	dateCount++;
+	var averageMealsPerDay = totalMeals / dateCount;
+	console.log('On day ' + dateCount + ', the lion had an average of ' 
+		+ averageMealsPerDay + ' meals per day!');
+	if (averageMealsPerDay < 4) {
+		tooHungryDay = dateCount;
+		console.log('The caretaker got eaten on day ' + tooHungryDay);
+		return;
```

Comment:
you'll want to use break here instead of return; break stops execution of a loop. return stops execution of a function.